File: conditionals/time.py
#santiago lagarde, Time of Day

import time
#first instance of time on programming
#print(time.gmtime())

#current time in seconds since gmtime

current =time.time()

#current time as used to seeing
now = time.ctime(current)
print(now)

#get just hour
local_time= time.localtime(current)
print(local_time)
hours = local_time.tm_hour

minutes = local_time.tm_min

print(minutes)

print(hours)

File: conditionals/time.c
//santiago L, c time

#include <stdio.h> 
#include <time.h>

int main(void){
    //time since jan 1st 1970
    //time_t seconds;

    //seconds = time(NULL);
    //printf("seconds since jan 1st, 1970 - %d\n", seconds);
   

    //current time
    time_t rawtime;
    struct tm * timeinfo;
    time(&rawtime);
    timeinfo = localtime(&rawtime);
    printf("current time and date is %s", asctime(timeinfo));
    
    //curent hour
    time_t now = time(NULL);
    struct tm *tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);
    return 0;

}
